docs/app/components/drawer/drawer-demo.component.ts

```diff
@@ -2,7 +2,8 @@ import { Component } from '@angular/core';
 
 @Component({
     selector: 'hc-sidenav-demo',
-    templateUrl: './drawer-demo.component.html'
+    templateUrl: './drawer-demo.component.html',
+    styles: [ '[hc-button] { margin-top: 10px; margin-right: 10px;}' ]
```

Comment:
this is repeated quite a few places, would it make sense to add it to the `styles.scss` global demo styles or even make a reusable css class?

Reply:
Yup, agreed - I realized this after I submitted the PM.  I'll move this into the global demo styles.